### PR TITLE
Default behavior when capture not specified

### PR DESCRIPTION
**Pull request by sunny36**
Added a little info when capture is not specified when creating a charge.

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -484,9 +484,9 @@ en:
       token ID. Once the token is used the card is attached to its new owner
       and the token is revoked and can't be used anymore.
     notice: |
-      **Full Credit Card data should never go through your server.**  
-      Do not send the credit card data to Omise from your servers directly.  
-      You must send the card data from the client browser via Javascript (Omise-JS).
```
Comment (fred) [inline locales/en.yml]: @sunny36  The two spaces at the end was there in purpose. it adds a new brake line.
did you remove in purpose?

Comment (sunny36) [inline locales/en.yml]: Not internationally, my editor must have removed all the trailing white space.  I'll add it back.

Comment (fred) [inline locales/en.yml]: Right.
I disabled auto-remove of trailing spaces on my editor just because of Markdown.

**Comment by robinclart** [comment]
:+1:

**Pull request merged**